WordPressKit/WordPressComServiceRemote+SiteSegments.swift

```diff
@@ -84,34 +84,16 @@ public enum SiteSegmentsResult {
     case failure(SiteSegmentsError)
 }
 
-public struct SiteSegmentsRequest: Encodable {
```

Comment:
Incidentally, I created [this issue](https://github.com/wordpress-mobile/WordPressKit-iOS/issues/72) to address the apparent duplication of the `locale` parameter.